lib/ui/settings/widgets/repository_selector/repository_selector_view.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:gitdone/core/models/repository_details.dart';
+import 'package:provider/provider.dart';
+
+import 'repository_selector_view_model.dart';
+
+class RepositorySelector extends StatefulWidget {
+  const RepositorySelector({super.key});
+
+  @override
+  State<RepositorySelector> createState() => _RepositorySelectorState();
+}
+
+class _RepositorySelectorState extends State<RepositorySelector> {
+  DropdownMenuItem<RepositoryDetails> convertRepoToItem(
+    RepositoryDetails repo,
+  ) {
+    return DropdownMenuItem(
+      value: repo,
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.start,
+        children: [
+          Padding(
+            padding: const EdgeInsets.fromLTRB(0, 0, 0, 8),
+            child: Image.network(repo.avatarUrl, width: 24, height: 24),
+          ),
+          Padding(
+            padding: EdgeInsets.fromLTRB(0, 0, 0, 8),
+            child: Text(repo.name),
+          ),
+          Text("(${repo.owner})", style: TextStyle(color: Colors.grey)),
+        ],
```

Comment:
There is no padding between the image and Repo name, and the owner text is moved to the bottom:
![image](https://github.com/user-attachments/assets/d1b53b80-f6f5-44c6-9216-5b51c436fc20)

Maybe this would look better (not tested):
```suggestion
        children: [
          Image.network(repo.avatarUrl, width: 24, height: 24),
          const Padding(padding: EdgeInsets.all(8.0)),
          Text(repo.name),
          const Padding(padding: EdgeInsets.all(8.0)),
          Text("(${repo.owner})", style: TextStyle(color: Colors.grey)),
        ],
```